src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -41,7 +46,9 @@ public class LogicManagerTest {
     public Path temporaryFolder;
 
     private Model model = new ModelManager();
+    private NewModel newModel = new NewModelManager();
```

Comment:
@mounilsankar are we creating a NewLogicManagerTest or are we changing this one?

Reply:
done